task-list/app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,53 @@
+class TasksController < ApplicationController
+
+
+  def index   #this is enough for the controller to render a view, but need to tell rails when this is an appropriate action
```

Comment:
Hey, I'm not sure what your comment means here. I'm guessing it's a reminder for yourself. For best practice, I'd start assuming that comments are for other people to understand your code, so you want to be clear in them.